.github/workflows/ci-docs.yml

```diff
@@ -71,19 +71,21 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
+          persist-credentials: false
       - run: mkdir ./bin
       - run: curl -sfL https://raw.githubusercontent.com/traefik/structor/master/godownloader.sh | bash -s -- -b ./bin ${STRUCTOR_VERSION}
       - run: sudo ./bin/structor -o janusgraph -r janusgraph 
             --force-edit-url 
-            --rqts-url="https://raw.githubusercontent.com/janusgraph/janusgraph/master/docs/requirements.txt" 
-            --dockerfile-url="https://raw.githubusercontent.com/janusgraph/janusgraph/master/docs/docs.Dockerfile" 
-            --dockerfile-name="default-docs.Dockerfile" 
-            --menu.js-url="https://raw.githubusercontent.com/janusgraph/janusgraph/master/docs/theme/structor-menu.js.gotmpl" 
+            --rqts-url="https://raw.githubusercontent.com/JanusGraph/janusgraph/master/requirements.txt" 
+            --dockerfile-url="https://raw.githubusercontent.com/JanusGraph/janusgraph/master/docs.Dockerfile" 
+            --dockerfile-name="https://raw.githubusercontent.com/JanusGraph/janusgraph/master/docs.Dockerfile" 
```

Comment:
Just a question, `dockerfile-name` should be the same as `dockerfile-url`?

Reply:
Their is bug inside of structor.

Reply:
Got it. Thanks!